addons/storyshots/src/test-bodies.js

```diff
@@ -7,6 +7,13 @@ export const snapshotWithOptions = options => ({
   renderTree,
   snapshotFileName,
 }) => {
+  if (options.snapshotSerializers) {
+    options.snapshotSerializers.forEach(serializer => {
```

Comment:
The serializers will be added during the test. Is it ok? I think it's not the best place to set them

Reply:
yeah, this is not ideal. i've moved serializer addition it to the initialization function